### ISSUE TITLE
SlevomatCodingStandard.Functions.RequireTrailingCommaInCall doesn't report missing commas if the last param is array function

### ISSUE THREAD

**Issue by sampsasaarela** [slevomat/coding-standard#1652]
Seems that `SlevomatCodingStandard.Functions.RequireTrailingCommaInCall` doesn't report missing commas if the last param is an array function.

For example, this doesn't report error:
```php
$this->myFunction(
    $channel,
    'request',
    fn ($m) => $m['param]
);
```

but this report

```php
$this->myFunction(
    $channel,
    'request',
);

$this->myFunction(
    $channel,
    'request',
    function () {
     // test
    }
);
```

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/8d0f603befc9dbc50735dc5cb4ef62015a6d915b
